robottelo/ui/settings.py

```diff
@@ -13,6 +13,27 @@ def navigate_to_entity(self):
         """Navigate to Settings entity page"""
         Navigator(self.browser).go_to_settings()
 
+    def remove_parameter(self, tab_locator, param_name):
+        """Removes the value of selected parameter under settings"""
+        if self.wait_until_element(tab_locator) is None:
```

Comment:
that error handling is pretty much redundant as `click` method perform it automatically